internal/providers/terraform/hcl_provider.go

```diff
@@ -390,22 +494,35 @@ type ResourceData struct {
 	Type              string                 `json:"type"`
 	Name              string                 `json:"name"`
 	ProviderConfigKey string                 `json:"provider_config_key"`
-	Expressions       map[string]interface{} `json:"expressions"`
+	Expressions       map[string]interface{} `json:"expressions,omitempty"`
+	SchemaVersion     int                    `json:"schema_version"`
+	CountExpression   *countExpression       `json:"count_expression,omitempty"`
 }
 
 type ModuleCall struct {
-	Source string           `json:"source"`
-	Module ModuleCallModule `json:"module"`
-}
-
-type ModuleCallModule struct {
-	Resources []ResourceData `json:"resources"`
+	Source string       `json:"source"`
+	Module ModuleConfig `json:"module"`
```

Comment:
Should we call it `ModuleConfig`?
```suggestion
	ModuleConfig ModuleConfig `json:"module"`
```

There are too many "module" words thrown around. :) I guess types help in the editor, but it requires extra attention when reading in GH diff. How do you feel about it?

Reply:
Yeah sounds good